reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Running filtered test: " + testId.join(", ") +
+			" <a id='qunit-clearFilter' href='" +
+			setUrl({ filter: undefined, module: undefined, testId: undefined }) +
+			"'>" + "Clear filter" + "</a></div>";
```

Comment:
Ok, changing that-- @leobalter and figured "Clear filter" made sense given the function's name, but "Run all tests" is more straightforward.